Microsoft.Toolkit.Uwp.UI.Controls.Design/Controls/WrapPanel.Typedata.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
+{
+#if VS_DESIGNER_PROCESS_ISOLATION
+    internal static partial class ControlTypes
+    {
+        internal static readonly Type WrapPanel = typeof(WrapPanel);
+    }
+#else
+    internal static partial class ControlTypes
+    {
+        internal const string WrapPanel = RootNamespace + "." + nameof(WrapPanel);
+    }
+
+    internal static class WrapPanel
```

Comment:
This is just a manual shadow copy to avoid typing in strings, eh? Manually maintaining these seems like a pain. Since we're only using them the one time, the strings don't seem that big of an issue.

I think if we wanted to do this, we could look at adding a generator in the future to do this automatically, but that doesn't have to be part of this initial scope.

Reply:
> ...Manually maintaining these seems like a pain...    ...adding a generator in the future to do this automatically...

I already have a separate branch working on CodeGen. This was generated from a tool written for that very purpose. I'll open a PR as soon as it's ready.